.editorconfig

```diff
@@ -0,0 +1,16 @@
+root = true
```

Comment:
Sure! [EditorConfig](http://editorconfig.org/) is really good for keeping line endings, indentation, trailing spaces and so on and so forth in sync between different IDEs. It's just a quick headache reducer as long as contributors have it installed in their editors (some of which have it bundled out of the box).